src/drawing/DrawingAnnotations.tsx

```diff
@@ -86,6 +88,14 @@ const DrawingAnnotations = (props: Props): JSX.Element => {
         undoDrawingPathGroup();
     };
 
+    React.useEffect(() => {
+        const { current: popup } = popupDrawingToolbarRef;
+
+        if (popup?.popper) {
```

Comment:
Should we check for drawnPathGroups.length? What happens if we undo until drawnPathGroups is empty?